react/localState/index.js

```diff
@@ -99,14 +105,33 @@ export default function(client) {
       updateOrderFormCheckin: replayOrderFormServerMutation(
         updateOrderFormCheckin
       ),
+      updateItemsSentToServer: (_, __, { cache }) => {
+        const query = minicartItemsQuery
+        const {
+          minicart: { items: prevItems },
+        } = cache.readQuery({ query })
+        const prevItemsParsed = JSON.parse(prevItems)
```

Comment:
just to be sure, are these JSON guaranteed to be valid?

Reply:
yes hehe